Documentation/common-issues.md

```diff
@@ -458,6 +459,36 @@ rbd
 ```
 Now when a host is restarted, the module should be loaded automatically.
 
+# Rook Agent rbd module missed error
+## Symptoms
+* Rook Agent in `Error` status when deploying the Rook operator with `kubectl create -f operator.yaml`:
+```
+$kubectl -n rook-ceph-system get pod
+NAME                                 READY     STATUS    RESTARTS   AGE
+rook-ceph-agent-gfrm5                0/1       Error     0          14s
+rook-ceph-operator-5f4866946-vmtff   1/1       Running   0          23s
+rook-discover-qhx6c                  1/1       Running   0          14s
+```
+* Rook Agent logs contain below messages:
+```
+2018-08-10 09:09:09.461798 I | exec: Running command: cat /lib/modules/4.15.2/modules.builtin
+2018-08-10 09:09:09.473858 I | exec: Running command: modinfo -F parm rbd
+2018-08-10 09:09:09.477215 N | ceph-volumeattacher: failed rbd single_major check, assuming it's unsupported: failed to check for rbd module single_major param: Failed to complete 'check kmod param': exit status 1. modinfo: ERROR: Module rbd not found.
+2018-08-10 09:09:09.477239 I | exec: Running command: modprobe rbd
+2018-08-10 09:09:09.480353 I | modprobe rbd: modprobe: FATAL: Module rbd not found.
+2018-08-10 09:09:09.480452 N | ceph-volumeattacher: failed to load kernel module rbd: failed to load kernel module rbd: Failed to complete 'modprobe rbd': exit status 1.
+failed to run rook ceph agent. failed to create volume manager: failed to load kernel module rbd: Failed to complete 'modprobe rbd': exit status 1.
+```
+
+## Solution
+From the log message of Agent, we can see that the `rbd` kernel module is missed in the current system, no matter it's a builtin or a loadable external kernel module.
+In this case, you have two alternatives to address this issue:
+
+* Re-configure your kernel to make sure the `CONFIG_BLK_DEV_RBD=y` in the `.config` file, then build the kernel again which will make the `rbd` as a builtin kernel module.
+* Re-configure your kernel to make sure the `CONFIG_BLK_DEV_RBD=m` in the `.config` file, then build the kernle again which will make the `rbd` as an exteranl loadable kernel module.
+
+Switch to the new built kernel in your system, this issue should be fixed: the Agent will be in normal `running` status if everthing is OK.
```

Comment:
Maybe adding some hints, e.g. rebooting the system to use the new kernel would be good.

`is OK` -> `was done correctly`

Reply:
Could you please change `is OK` to `was done correctly`